simpleclient/src/main/java/io/prometheus/client/exemplars/ExemplarConfig.java

```diff
@@ -0,0 +1,79 @@
+package io.prometheus.client.exemplars;
+
+/**
+ * Static configuration class for Exemplar behavior.
+ */
+public class ExemplarConfig {
+
+  private static final ExemplarConfig instance = new ExemplarConfig();
+
+  private final ExemplarSampler noopExemplarSampler;
+  private final ExemplarSampler defaultExemplarSampler;
```

Comment:
Shouldn't these be static? They aren't tied to an instance.

Reply:
Made them static.